src/locales/cs/translation.json

```
@@ -1,82 +1,82 @@
-        "gold_many": "{{count}} gold",
-        "gold_other": "{{count}} gold",
-        "card_one": "{{count}} card",
-        "card_few": "{{count}} cards",
-        "card_many": "{{count}} cards",
-        "card_other": "{{count}} cards",
-        "card_witchertrophy_one": "{{count}} Witcher Trophy Card",
-        "card_witchertrophy_few": "{{count}} Witcher Trophy Cards",
-        "card_witchertrophy_many": "{{count}} Witcher Trophy Cards",
-        "card_witchertrophy_other": "{{count}} Witcher Trophy Cards",
-        "card_witchermagetrophy_one": "{{count}} Witcher/Mage Trophy Card",
-        "card_witchermagetrophy_few": "{{count}} Witcher/Mage Trophy Cards",
-        "card_witchermagetrophy_many": "{{count}} Witcher/Mage Trophy Cards",
-        "card_witchermagetrophy_other": "{{count}} Witcher/Mage Trophy Cards",
-        "token_level1_one": "{{count}} Level I token",
-        "token_level1_few": "{{count}} Level I tokens",
-        "token_level1_many": "{{count}} Level I tokens",
-        "token_level1_other": "{{count}} Level I tokens",
-        "token_shield_one": "{{count}} Shield Token",
-        "token_shield_few": "{{count}} Shield Tokens",
-        "token_shield_many": "{{count}} Shield Tokens",
-        "token_shield_other": "{{count}} Shield Tokens",
-        "underConstruction": "Under Construction!"
+        "expansions": "Rozšíření",
+        "Step": "Krok",
+        "players": "Hráči",
+        "player1": "Hráč 1",
+        "player2": "Hráč 2",
+        "player3": "Hráč 3",
+        "player4": "Hráč 4",
+        "player5": "Hráč 5",
+        "gold_one": "{{count}} zlaťák",
+        "gold_few": "{{count}} zlaťáky",
+        "gold_many": "{{count}} zlaťáku",
+        "gold_other": "{{count}} zlaťáků",
+        "card_one": "{{count}} karta",
+        "card_few": "{{count}} karty",
+        "card_many": "{{count}} karty",
+        "card_other": "{{count}} karet",
+        "card_witchertrophy_one": "{{count}} karta trofeje zaklínače",
+        "card_witchertrophy_few": "{{count}} karty trofeje zaklínače",
+        "card_witchertrophy_many": "{{count}} karty trofeje zaklínače",
+        "card_witchertrophy_other": "{{count}} karet trofeje zaklínače",
+        "card_witchermagetrophy_one": "{{count}} karta trofeje zaklínače/mága",
+        "card_witchermagetrophy_few": "{{count}} karty trofeje zaklínače/mága",
+        "card_witchermagetrophy_many": "{{count}} karty trofeje zaklínače/mága",
+        "card_witchermagetrophy_other": "{{count}} karet trofeje zaklínače/mága",
+        "token_level1_one": "{{count}} žeton úrovně I",
+        "token_level1_few": "{{count}} žetony úrovně I",
+        "token_level1_many": "{{count}} žetonu úrovně I",
+        "token_level1_other": "{{count}} žetonů úrovně I",
+        "token_shield_one": "{{count}} žeton Štítu",
+        "token_shield_few": "{{count}} žetony Štítu",
+        "token_shield_many": "{{count}} žetonů Štítu",
+       "token_shield_other": "{{count}} žetonu Štítu",
```

Nit: this indent needs to be in line with the rest of the keys

```suggestion
        "token_shield_other": "{{count}} Shield Tokens",
```